gensim/models/translation_matrix.py

```diff
@@ -209,20 +211,20 @@ def translate(self, source_words, topn=5, additional=None, source_lang_vec=None,
         # if the language word vector not provided by user, use the model's
         # language word vector as default
         if source_lang_vec is None:
-            warnings.warn("the parameter source_lang_vec didn't specified,"
-                          " use the model's source language word vector as default")
+            warnings.warn("The parameter source_lang_vec didn't specified, use the model's source language word vector as default")
             source_lang_vec = self.source_lang_vec
 
         if target_lang_vec is None:
-            warnings.warn("the parameter target_lang_vec isn't specified,"
-                          " use the model's target language word vector as default")
+            warnings.warn("The parameter target_lang_vec isn't specified, use the model's target language word vector as default")
             target_lang_vec = self.target_lang_vec
 
         # if additional is provided, bootstrapping vocabulary from the source language word vector model.
-        if additional is not None:
+        if gc:
+            if additional is None:
+                raise RuntimeError("When using the globally corrected neighbour retrieval method, the additional parameter which the number of words sampled from source space must be provided.")
```

Comment:
There seems to be a word missing after `which` -- the sentence doesn't make sense now.

Reply:
Thanks, I will revise it. And if there is any other inappropriate expression, please point out.